app/src/main/java/org/wikipedia/suggestededits/SuggestedEditsTasksFragment.kt

```diff
@@ -53,21 +63,110 @@ class SuggestedEditsTasksFragment : Fragment() {
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
         super.onViewCreated(view, savedInstanceState)
         (activity as AppCompatActivity).supportActionBar!!.elevation = 0f
+        //Todo: remove after review
+        dummyButtons()
```

Comment:
Not sure what it is.

Reply:
Oh I have just created dummy buttons so Robin can test the disabled state and paused state... just for temporary purpose...